Source/Dafny/DafnyOptions.cs

```diff
@@ -101,6 +101,11 @@ public enum IncludesModes { None, Immediate, Transitive }
     public int Allocated = 3;
     public bool UseStdin = false;
     public bool ShowSnippets = false;
+    public enum TestModes { None, Block, Path };
```

Comment:
Can you consider moving the code that was added to `DafnyOptions` to a class like `TestGenerationOptions` that includes the fields, the parsing logic and documentation, and then reference that from `DafnyOptions` ?